Example-Crypto-REXX/README.md

```diff
@@ -0,0 +1,104 @@
+## Example-Crypto-REXX
+
+This sample queries Image Activation profiles to list crypto attributes for LPARs. It stores the results, which are in .csv format, in a z/OS data set member.  You can import the content into Microsoft™ Excel or another application of your choice.
+
+This sample uses **HWIREST API** to do the following:
+- List CPCs and retrieve the URI and target name associated with the LOCAL CPC or the *CPCname* provided
+- List the LPARs on the designated CPC and query the LPAR status.
```

Comment:
this says LPARs but the description is specifically talking about image activation profiles?

Reply:
Crypto information is in the image activation profile, but I am retrieving it only for LPARs in the requested state.  The exec is getting the LPAR status, and then querying the image activation profile.  I will try to clarify this section.